gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -245,6 +245,7 @@ typedef struct
 {
   unsigned int num_tensors; /**< The number of tensors */
   GstTensorInfo info[NNS_TENSOR_SIZE_LIMIT]; /**< The list of tensor info */
+  GstTensorInfo *extra; /**< Info for tensors whose idx is larger than NNS_TENSOR_SIZE_LIMIT */
```

Comment:
The description of info is as follows.
GstTensorInfo info[NNS_TENSOR_SIZE_LIMIT]; /**< The list of tensor info */

How about using 'list' instead of 'idx'?

Reply:
Thanks for this suggestion, I used the word 'list'